sci-astronomy/stellarium/stellarium-0.18.2.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils desktop flag-o-matic gnome2-utils virtualx
+
+DESCRIPTION="3D photo-realistic skies in real time"
+HOMEPAGE="http://www.stellarium.org/"
+SRC_URI="
+	https://github.com/Stellarium/stellarium/releases/download/v${PV}/${P}.tar.gz
+	stars? (
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_4_1v0_2.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_5_2v0_1.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_6_2v0_1.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_7_2v0_1.cat
+		https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_8_2v0_1.cat
+	)"
+
+PATCHES=( "${FILESDIR}"/${P}.patch )
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug gps media nls stars test"
+
+RDEPEND="
+	media-fonts/dejavu
```

Comment:
Please keep deps sorted lexicographically (unconditional above conditional, check also following lines). Also, is the build system checking for dejavu? It may be an `RDEPEND` only.

Reply:
I have no idea why dejavu is here, it's already the case in stellarium-0.16.0-r1.ebuild
It's already in RDEPEND

I'll reorder, sure.